packages/compiler/test/server/get-hover.test.ts

```diff
@@ -394,6 +394,57 @@ describe("compiler: server: on hover", () => {
         },
       });
     });
+
+    it("model with extends and is (full definition expected)", async () => {
+      const hover = await getHoverAtCursor(
+        `
+          @service(#{title: "RT"})
+          namespace TestNs;
+          
+          model Do┆g is Animal<string, DogProperties> {
+              barkVolume: int32;
+          }
+
+          model Animal<T, P> extends AnimalBase<P>{
+              name: string;
+              age: int16;
+              tTag: T;
+          }
+
+          model AnimalBase<P> {
+              id: string;
+              properties: P;
+          }
+
+
+          model DogProperties {
+              breed: string;
+              color: string;
+          }
+        `,
+      );
+      deepStrictEqual(hover, {
+        contents: {
+          kind: MarkupKind.Markdown,
+          value: `\`\`\`typespec
+model TestNs.Dog
+\`\`\`
+
+*Full Definition:*
+
+\`\`\`typespec
+model TestNs.Dog{
```

Comment:
```suggestion
model TestNs.Dog {
```
think missing a space here